src/main/java/nextstep/subway/domain/LineRepository.java

```diff
@@ -0,0 +1,8 @@
+package nextstep.subway.domain;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface LineRepository extends JpaRepository<Line, Long> {
+}
```

Comment:
```suggestion
}

```

여러 버전의 유닉스 OS간의 공통된 API 및 인터페이스를 정의해놓은 POSIX 명세 중 text file 부분을 보면 유닉스 시스템은 text file들을 서로 구분할때 EOL을 사용하기 때문에 파일을 만들때마다 파일의 끝에 EOL을 넣어주는 것이 안전합니다~

Reply:
오,,, 제가 몰랐던 사실입니다 참고하겠습니다 👍